x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -209,13 +209,9 @@ export const isolationRequestHandler = function (
     }
 
     try {
-      let esClient = context.core.elasticsearch.client.asCurrentUser;
-      if (doesLogsEndpointActionsDsExist) {
-        // create action request record as system user with user in .fleet-actions
-        esClient = context.core.elasticsearch.client.asInternalUser;
-      }
-      // write as the current user if the new indices do not exist
-      // <v7.16 requires the current user to be super user
+      const esClient = context.core.elasticsearch.client.asInternalUser;
+      // write as the internal user if the new indices do not exist
+      // 8.0+ requires internal user to write to system indices
```

Comment:
This is okay to mention here 👍

Reply:
Although why do we need to backport this to 7.16? If this is only for `>=v8.0.0`

Reply:
Resolved offline, this is to support rolling upgrades without breaking the isolate host feature.